tests/functional/fx_desktop_handshake.js

```diff
@@ -241,7 +237,7 @@ define([
 
     'Sync signin page - no user signed into browser, user signed in locally': function () {
       return this.remote
-        // First, sign in the user to populate localStorage
+      // First, sign in the user to populate localStorage
         .then(openPage(SIGNIN_PAGE_URL, selectors.SIGNIN.HEADER, {
```

Comment:
...and these comments.

Reply:
I've seen this when using WebStorm, it formats things unexpectedly.